test/js_interop_helpers_test/shared_tests.dart

```diff
@@ -50,7 +58,7 @@ void sharedJsFunctionTests() {
 
     group('markChildValidated', () {
       test('is function that does not throw when called', () {
-        expect(() => markChildValidated(new EmptyObject()), returnsNormally);
+        expect(() => markChildValidated(newObject()), returnsNormally);
```

Comment:
This test should likely remain unchanged unless `markChildValidated` is backwards-compatible to still work with `EmptyObject` - in which case a test case for both should be maintained.

Reply:
sure

Reply:
see https://github.com/cleandart/react-dart/pull/160#discussion_r262942145